libraries/shared/watcher/event_watcher.go

```diff
@@ -128,6 +136,25 @@ func (watcher *EventWatcher) withRetry(call func() error, expectedErrors []error
 	}
 }
 
+func addStatusForHealthCheck(msg []byte) error {
```

Comment:
Wondering if we should move this to a `utils.go` file or something since it's being used by both the `EventWatcher` and the `StorageWatcher` 🤔